docs/development/release.md

```diff
@@ -10,6 +10,16 @@ Read more about the semantic versioning [here](https://docs.npmjs.com/getting-st
 
 ### 1. Create a branch for the new release
 
+#### From Master
+
+If creating a new minor version release from master, create a branch titled `release-1.MINOR.x` (e.g. release-1.14.x) as a feature branch from master to allow us to easily backport patches to that version.  Then create a release branch from it.
+
+The name is not important, but `release-x.y.z` seems like a good choice.
+
+#### Backporting
+
+Checkout the release-1.YY.x branch, then create a release branch from it.
```

Comment:
`Checkout the release-1.YY.x branch`
Does this assume that branch already exists somehow? Or is it meant to be checked out from the version tag to which you are backporting?

Reply:
I realize that if the `From Master` instructions were previously followed this branch will exist. Is that always the case?

Reply:
Excellent point.  I would expect it would always be the case, if not it should be inferred from the instructions.  I updated a bit to clarify